src/version.c

```diff
@@ -20,69 +20,14 @@
 #define STR_EXPAND(x) #x
 #define STR(x) STR_EXPAND(x)
 
-void
-version_get_info(VersionInfo *vinfo)
-{
-	memset(vinfo, 0, sizeof(VersionInfo));
-	vinfo->version[0] = strtol(TIMESCALEDB_MAJOR_VERSION, NULL, 10);
-	vinfo->version[1] = strtol(TIMESCALEDB_MINOR_VERSION, NULL, 10);
-	vinfo->version[2] = strtol(TIMESCALEDB_PATCH_VERSION, NULL, 10);
-
-	if (strlen(TIMESCALEDB_MOD_VERSION) > 0)
-	{
-		StrNCpy(vinfo->version_mod, TIMESCALEDB_MOD_VERSION, sizeof(vinfo->version_mod));
-		vinfo->has_version_mod = true;
-	}
-}
-
-/*
- * Compare two versions.
- *
- * It returns an integer less than, equal to, or greater than zero if version
- * v1 is found, respectively, to be less than, to match, or be greater than
- * version v2.
- */
-int
-version_cmp(VersionInfo *v1, VersionInfo *v2)
-{
-	int			i;
-
-	for (i = 0; i < 3; i++)
-	{
-		if (v1->version[i] > v2->version[i])
-			return 1;
-
-		if (v1->version[i] < v2->version[i])
-			return -1;
-	}
-
-	/*
-	 * Note that the version mod signifies a pre-release version, so having a
-	 * version mod is "less" than not having one with otherwise identical
-	 * versions
-	 */
-	if (v1->has_version_mod && !v2->has_version_mod)
-		return -1;
-
-	if (!v1->has_version_mod && v2->has_version_mod)
-		return 1;
-
-	/* Compare the version mod lexicographically */
-	if (v1->has_version_mod && v2->has_version_mod)
-		return strncmp(v1->version_mod, v2->version_mod, sizeof(v1->version_mod));
-
-	return 0;
-}
-
 bool
-version_parse(const char *version, VersionInfo *result)
+version_parse(const char *version)
```

Comment:
this should be named something like version_validate now

Reply:
Do we even need this if we remove the version parsing?

Reply:
removed